crates/viewer/re_space_view_graph/src/graph/index.rs

```diff
@@ -0,0 +1,35 @@
+use re_log_types::{EntityPath, EntityPathHash};
+use re_types::datatypes;
+
+use super::GraphNodeHash;
+
+#[derive(Clone, Copy, PartialEq, Eq)]
+pub(crate) struct NodeIndex {
+    pub entity_hash: EntityPathHash,
+    pub node_hash: GraphNodeHash,
+}
+
+impl nohash_hasher::IsEnabled for NodeIndex {}
+
+impl std::hash::Hash for NodeIndex {
+    fn hash<H: std::hash::Hasher>(&self, state: &mut H) {
+        // TODO(grtlr): Consider using `write_usize` here, to further decrease the risk of collision.
+        let combined = self.entity_hash.hash64() << 32 | self.node_hash.hash64();
+        state.write_u64(combined);
+    }
+}
```

Comment:
Same, can't this be derived?

Reply:
The implementation here is non-standard, because `nohash_hasher::IsEnabled` requires a single cals to `write_{u64|usize|...}` as per the [crate documentation](https://crates.io/crates/nohash-hasher).

Reply:
Ah ok. Please add a comment stating that then.